README.md

```diff
@@ -102,8 +105,11 @@ An example of capturing location data is available in the example app.
 
 ###Contacts Syncing###
 
-Shine can capture a device's contacts if the host app has the required contacts access permissions.
+Shine can capture a device's contacts if the host app has the required contacts access permissions. Make a call to `[Shine syncContacts]` after permission to access contacts has been granted.
+
 ```objective-c
```

Comment:
Worth mentioning that if contact access is already granted they don't have to do anything? In case they have a load of people with contacts enabled and end up putting it in appDidFinishWithOptions because they think they have to implement it for people who already have contacts enabled?